compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -376,20 +376,29 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             // If there are two targets (one conditional, one fallback), emit `br` instead of
             // `switch`.
             let (test_value, target) = target_iter.next().unwrap();
+            let otherwise = targets.otherwise();
+            let true_cold = self.cold_blocks[target];
+            let false_cold = self.cold_blocks[otherwise];
+            let expect = if true_cold != false_cold { Some(false_cold) } else { None };
             let lltrue = helper.llbb_with_cleanup(self, target);
-            let llfalse = helper.llbb_with_cleanup(self, targets.otherwise());
+            let llfalse = helper.llbb_with_cleanup(self, otherwise);
```

Comment:
Could we take the opportunity to rename those to `lltarget` and `llotherwise`?

Reply:
Renamed